languages/elixir/exercises/concept/try-rescue-else-after/.docs/introduction.md

```diff
@@ -0,0 +1,64 @@
+## Errors Expanded
+
+Using `try..rescue` is a powerful construct for catching errors when they occur in programs so that function can return sensible values if it is required. The `try..rescue` construct also offers us two additional features we can make use of:
+
+- the `else` block
+  - When the try block succeeds, the result is matched to this block.
+
+```elixir
+try do
+  :a
+rescue
+  _ -> :error
+else
+  :a -> :success
+end
+# => :success
+```
+
+- the `after` block
+  - Whether the try block succeeds or raises, the code in the `after` block is always executed as long as the process is alive at that point.
```

Comment:
We didn't explain processes yet, but we should be able to come up with a process exercise that comes before this one in the exercise tree.

Reply:
Agreed, however process being ambiguous at this point isn't too bad since we haven't defined it at all, and don't define it here to mean something contradictory.